HelpSource/Guides/Internal-Snooping.schelp

```diff
@@ -57,38 +66,39 @@ Server.filenameSymbol.postln; // the path to the file that defined this class
 
 (
 // print all classes whose names start with 'F'
-Class.allClasses.do({ arg class;
-	if (class.name.asString.at(0) == $F, { class.name.postln; });
-})
+Class.allClasses.do { | class |
+	if (class.name.asString.at(0) == $F) { class.name.postln }
+}; "";
```

Comment:
I say let's not get into the habit of doing this in helpfiles. There's no way we can keep it consistent everywhere, and it adds noise to what is supposed to be a simple example. I'm assuming you're doing this so that users don't see `-> Class` in the post window. Making that behavior configurable would be a good ticket.